### PR TITLE
Allow specifying the fsname as a mount option.

### PR DESCRIPTION
**Pull request by andrewchambers**

### CODE REVIEW

File: src/s3ql/mount.py [hunk 1]
```diff
@@ -529,6 +533,8 @@ def compression_type(s):
                            'user and the root user.')
     parser.add_argument("--fg", action="store_true", default=False,
                       help="Do not daemonize, stay in foreground")
+    parser.add_argument("--fs-name", default='',
```
Comment (Nikratio) [inline src/s3ql/mount.py]: Let's make this `None` (same above)

File: src/s3ql/mount.py [hunk 2]
```diff
@@ -529,6 +533,8 @@ def compression_type(s):
...
+                      help="Mount name passed to fuse.")
```
Comment (Nikratio) [inline src/s3ql/mount.py]: Can we be more specific here? Eg "Will be shown by mount in the <foo> column" (I can't actually check where this goes right now)

**Comment by Nikratio** [comment]
Thanks! Looks reasonable. Please also add an entry to Changes.rst

File: src/s3ql/mount.py (hunk 2 above)
Comment (andrewchambers) [reply src/s3ql/mount.py]: Sure, that is where it goes btw.

**Pull request merged**